### PR TITLE
Add `sizes=auto` support info for `<img>` elements

### PR DESCRIPTION
**Pull request by tunetheweb**
#### Summary

The `sizes` attribute for an `` element now allows for the `auto` value:
https://github.com/whatwg/html/pull/8008

#### Test results and supporting details

Search for "auto" here: https://wpt.fyi/results/html/semantics/embedded-content/the-img-element/sizes/sizes-auto.html?label=experimental&label=master&aligned

Chrome issue: https://issues.chromium.org/issues/40862170#comment31
Firefox issue (not implemented yet): https://bugzilla.mozilla.org/show_bug.cgi?id=1819581
Safari issue (not implemented yet): https://bugs.webkit.org/show_bug.cgi?id=253143

### CODE REVIEW

File: api/HTMLImageElement.json [hunk 1]
```diff
@@ -1052,6 +1052,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "auto": {
+          "__compat": {
+            "description": "Auto-sizes for lazy-loaded images in HTML",
```
Comment (Elchi3) [inline api/HTMLImageElement.json:1058]:
```suggestion
            "description": "<code>auto</code> value",
```

**Review by Elchi3 — COMMENTED** [review]
I think this would be better in html/elements/img.json (or in both).

**Review by Elchi3 — COMMENTED** [review]
> Firefox issue (not implemented yet): https://bugzilla.mozilla.org/show_bug.cgi?id=1819581
> Safari issue (not implemented yet): https://bugs.webkit.org/show_bug.cgi?id=253143

Would you mind adding these as impl_url?

**Comment by tunetheweb** [comment]
> Would you mind adding these as impl_url?

Done!

> I think this would be better in html/elements/img.json (or in both).

Added to there too (as `sizes` is in both)

**Review by Elchi3 — APPROVED** [review]
Perfect, thank you! 👍

**Pull request merged**